webapp/components/survey/NodeDefsSelector/AttributesSelector/AttributeSelector.js

```diff
@@ -51,12 +52,14 @@ AttributeSelector.propTypes = {
   nodeDefUuidsAttributes: PropTypes.array.isRequired,
   onToggleAttribute: PropTypes.func.isRequired,
   showMultipleAttributes: PropTypes.bool,
+  itemLabel: PropTypes.func,
 }
 
 AttributeSelector.defaultProps = {
   canSelectAttributes: true,
   filterTypes: [],
   showMultipleAttributes: true,
+  itemLabel: NodeDef.getLabel,
```

Comment:
I would call this "itemLabelFunction"... otherwise if you see it among the props you think it's a string...